### PR TITLE
Install 'rsvg-convert'

### PR DESCRIPTION
**Pull request by mgeier**
See #78.

This is untested, I hope it works!

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -48,6 +48,9 @@ RUN apt-get -y install \
     latex-cjk-chinese-arphic-gbsn00lp latex-cjk-chinese-arphic-gkai00mp \
     fonts-symbola
 
+# Install 'rsvg-convert' for SVG -> PDF conversion
+RUN apt-get -y install librsvg2-bin
```
Comment (humitos) [inline Dockerfile]: I would add a comment here with this link also, https://github.com/missinglinkelectronics/sphinxcontrib-svg2pdfconverter

Comment (mgeier) [reply Dockerfile]: Done: 80f52ba4130c271f7b1fa94ff6e593b2d6ce0614

**Comment by mgeier** [comment]
Thanks for merging!

How is the release process of this image?

When will it be available on readthedocs.org?

**Comment by stsewd** [comment]
@mgeier here is some discussion about it https://github.com/rtfd/readthedocs-docker-images/pull/84

**Comment by humitos** [comment]
Hi all! `rsvg-convert` was installed in our latest testing image `5.0rc1` (not released yet). I'd like to know if you want to help us testing it. In that case, please let me know what are the projects you are interested on testing it and I will apply this Docker image for its building process.

**Comment by humitos** [comment]
Hrm... It failed, but with another error https://readthedocs.org/projects/nbsphinx/builds/8391747/

Any clue?

> BTW, I love nbsphinx ;)

**Comment by mgeier** [comment]
Hmm .. no idea. I'm wondering why `xpdf` is mentioned in the error ...

> BTW, I love nbsphinx ;)

Cool, thanks!

**Comment by mgeier** [comment]
@humitos I think there is actually a different issue involved, regarding auto-downloading of remote files. I'll investigate further an come back as soon as I know more ...

**Comment by mgeier** [comment]
Just a little follow-up:

> I think there is actually a different issue involved

It was indeed a different issue. A single SVG file was somehow broken, see https://github.com/spatialaudio/nbsphinx/pull/319

In the end `rsvg-convert` works great on RTD, thanks for making this possible!

**Comment by mgeier** [comment]
It looks like this has been reverted in #166.

Was this intentional?

**Comment by humitos** [comment]
@mgeier with the new `build.apt_packages` config file key, users can install APT packages https://docs.readthedocs.io/en/stable/config-file/v2.html#build-apt-packages. Due to that, this package is not required anymore to be installed by default.

**Pull request merged**